superset-frontend/src/dashboard/components/gridComponents/Tabs.jsx

```diff
@@ -89,6 +89,10 @@ const StyledTabsContainer = styled.div`
   .ant-tabs {
     overflow: visible;
 
+    .ant-tabs-nav-wrap {
+      min-height: ${({ theme }) => theme.gridUnit * 12.5}px;
```

Comment:
Nit, non-blocking: Why this `12.5` value here? Could we round it to a whole value?

Reply:
Hello @suddjian. Thanks for your feedback. This is to make it exactly 50px which should be the default height of the tabs component.

Reply:
If this is meant to match the tabs component, it would be nice to have them both reference the same constant value instead of magic numbers that happen to match up.